samples/connectedhomeip/lock/src/BoltLockManager.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#pragma once
+
+#include <cstdint>
+
+#include "AppEvent.h"
+
+struct k_timer;
+
+class BoltLockManager {
+public:
+	enum class Action : uint8_t { Lock, Unlock };
+
+	void Init();
+
+	bool IsUnlocked();
+	bool IsActionInProgress();
+
+	bool InitiateAction(Action action, bool chipInitiated);
+	bool CompleteCurrentAction(bool &chipInitiated);
+
+private:
+	enum class State : uint8_t { LockingInitiated = 0, LockingCompleted, UnlockingInitiated, UnlockingCompleted };
+
+	static constexpr uint32_t ACTUATOR_MOVEMENT_PERIOS_MS = 2000;
```

Comment:
Isn't there a rule to write const expressions using camelCase and MACRO_CASE is reserved for definitions?

Reply:
In CHIP some people use kConstantName and others CONSTANT_NAME. In nrf/samples consts are not distinguished anyhow and they're written as constant_name most of the time...

Reply:
on the other hand, enums in some samples follow the CONSTANT_NAME convention. Maybe I should leave ACTUATOR_MOVEMENT_PERIOD_MS and fix enums like State::LockingInitiated (should be State::LOCKING_INITIATED)?

Reply:
In my opinion using camelCase looks better. I don't know if we should follow CHIP examples convention in NCS, as I've seen in some cases using local variables with rather pythonic style names like: `variable_name`, so it seems that there is no one strong naming convention in CHIP.

Reply:
ok, but I meant that enums in NCS samples use CONSTANT_NAME. Not sure if camelCase is a common convention in NCS. At least for constants. I guess that is something we may talk about during the final review...